lib/plugins/jexlParser.js

```diff
@@ -136,32 +136,8 @@ function isTransform(identifier) {
 function contextAvailable(expression, context) {
     let error;
     try {
-        const lexer = jexl._getLexer();
-        const identifiers = lexer.tokenize(expression).filter(function (token) {
-            return token.type === 'identifier';
-        });
-        const keys = Object.keys(context);
         let validContext = true;
-        identifiers.some(function (element) {
-            if (!keys.includes(element.value) && !isTransform(element.value)) {
-                validContext = false;
-                logger.warn(
-                    logContext,
-                    'For expression "[%s]" context "[%j]" does not have element %s',
-                    expression,
-                    context,
-                    element.value
-                );
-            }
-            // FIXME: Workaround to allow keys of JSON geopoints
-            if (element.value === 'type' || element.value === 'coordinates') {
-                validContext = true;
-            }
-            return validContext === false;
-        });
-        if (validContext) {
-            jexl.evalSync(expression, context);
-        }
+        jexl.evalSync(expression, context);
         return validContext;
```

Comment:
Maybe better?

```suggestion
        return true;
```

And remove the `let validContext = true;` above